benchmark/README.md

```diff
@@ -280,8 +288,10 @@ mean       median     min        input size   model
 153.89     153.96     153.43     [224, 224]   MPPersonDet with ['person_detection_mediapipe_2023mar.onnx']
 44.29      44.03      43.62      [128, 256]   YoutuReID with ['person_reid_youtu_2021nov.onnx']
 91.28      92.89      89.79      [256, 256]   MPPose with ['pose_estimation_mediapipe_2023mar.onnx']
-254.78     256.13     245.60     [640, 480]   DB with ['text_detection_DB_IC15_resnet18_2021sep.onnx']
-254.98     255.20     245.60     [640, 480]   DB with ['text_detection_DB_TD500_resnet18_2021sep.onnx']
+427.53     428.67     425.63     [640, 480]   PPOCRv3DB with ['text_detection_ch_ppocrv3_2023may.onnx']
+427.79     429.28     425.63     [640, 480]   PPOCRv3DB with ['text_detection_en_ppocrv3_2023may.onnx']
+414.07     429.46     387.26     [640, 480]   PPOCRv3DB with ['text_detection_ch_ppocrv3_2023may_int8.onnx']
+406.10     407.83     383.41     [640, 480]   PPOCRv3DB with ['text_detection_en_ppocrv3_2023may_int8.onnx']
```

Comment:
All the backends are faster with the new model, only FP16 is slower than before.